packages/database/src/migrations/20200914043218-AddLaosSchoolsSchoolIndicatorsEiEDevelopmentPartnerSupportMapOverlayOtherResponse-modifies-data.js

```diff
@@ -0,0 +1,350 @@
+'use strict';
```

Comment:
This migration is for adding `Other` response to `Development partner support` overlay in School Indicators EiE group:

> To add response of other to Development partner support: Other (SchCVD022l)